scripts/promote_staging_to_prod.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+checks=$(curl -s  "https://api.github.com/repos/chanzuckerberg/single-cell-data-portal/actions/runs?actor=czibuildbot" | jq -r '.workflow_runs[] | select(.display_title == "stage") | .conclusion' | head -n 1)
```

Comment:
This is almost identical to `Check if Latest Staging Post-Deploy Tests are Passing`. You could consolidate by moving it into it's own bash script that takes the -H parameter.